src/index.js

```diff
@@ -30,6 +31,7 @@ function App() {
   const [wrongAnswer, setWrongAnswer] = useState(false)
   const { question, answers } = flashcard
   const [failedAnswers, setFailedAnswers] = useState([])
+  const [displaySettings, setDisplaySettings] = useState(false)
```

Comment:
same thing for wording, related to difficulty

Reply:
done